src/components/PriceChart.tsx

```
@@ -37,472 +47,302 @@ const PriceChart: React.FC<PriceChartProps> = ({ assetName, timeFrame = '24h', o
-    };
-  }, [canvasRef, selectedTimeframe]);
+      case '30d': {
+        // Create approximately 6 evenly spaced ticks, plus start and end dates
+        const tickCount = 6;
+        const ticks: number[] = [];
+        for (let i = 0; i <= tickCount; i++) {
+          ticks.push(startMs + (i * (duration / tickCount)));
+        }
+        return ticks;
+      }
+      case 'all': // 90 days
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)